ui/cryptomaterial/dropdown.go

```diff
@@ -149,6 +151,17 @@ func (d *DropDown) SelectedIndex() int {
 	return d.selectedIndex
 }
 
+func (d *DropDown) SetConvertTextSize(fun func(unit.Sp) unit.Sp) {
+	d.convertTextSize = fun
+}
+
+func (d *DropDown) getTextSize(textSize unit.Sp) unit.Sp {
+	if d.convertTextSize == nil {
+		return textSize
+	}
+	return d.convertTextSize(textSize)
+}
+
```

Comment:
i don't see any conversion actually happening, am i missing something?

Reply:
It actually happens, but first you need set convert function for it